src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { buttonName } = props;
```

Comment:
- [ ] **(OPTIONAL)** You can make this code even shorter by simply destructuring the props directly in the function's argument :smile: